### PR TITLE
SUBMARINE-590. [WEB] Update chart/param/metric UI

### PR DESCRIPTION
**Pull request by pingsutw**
### What is this PR for?
- Improve chart/param/metric UI
- Fix the database connection error due to the update in SUBMARINE-575
- Read the data from the `metric` table, and draw charts in workbench

Following commands could reproduce the experiment
```bash
curl -X POST -H "Content-Type: application/json" -d '
{
  "meta": {
    "name": "tf-mnist2",
    "namespace": "submarine",
    "framework": "TensorFlow",
    "cmd": "python3 /var/example/tracking.py",
    "envVars": {
      "ENV_1": "ENV1"
    }
  },
  "environment": {
    "image": "pingsutw/tracking-example:0.5.0.dev"
  },
  "spec": {
    "Ps": {
      "replicas": 1,
      "resources": "cpu=1,memory=1024M"
    },
    "Worker": {
      "replicas": 1,
      "resources": "cpu=1,memory=1024M"
    }
  }
}
' http://127.0.0.1:8080/api/v1/experiment

```

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-590

### How should this be tested?
https://travis-ci.org/github/pingsutw/hadoop-submarine/builds/714396434

### Screenshots (if appropriate)
Screenshot from 2020-08-03 12-42-34

Screenshot from 2020-08-03 12-42-35

Screenshot from 2020-08-03 12-42-38

Screenshot from 2020-08-03 12-42-37

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by liuxunorg** [comment]
@pingsutw Please fix the files conflict. Thanks.

File: submarine-server/server-core/src/main/java/org/apache/submarine/server/experiment/ExperimentManager.java [hunk 1]
```diff
@@ -85,13 +87,33 @@ private ExperimentManager(Submitter submitter) {
...
+    experiment.setExperimentId(id);
+
+    spec.getMeta().getEnvVars().remove(RestConstants.JOB_ID);
+    spec.getMeta().getEnvVars().remove(RestConstants.SUBMARINE_TRACKING_URI);
     experiment.setSpec(spec);
     cachedExperimentMap.putIfAbsent(experiment.getExperimentId().toString(), experiment);
     return experiment;
   }
 
+  private String getSQLalchemyURL() {
```
Comment (jiwq) [inline submarine-server/server-core/src/main/java/org/apache/submarine/server/experiment/ExperimentManager.java:108]:
```suggestion
  private String getSQLAlchemyURL() {
```

File: submarine-server/server-core/src/main/java/org/apache/submarine/server/experiment/ExperimentManager.java [hunk 2]
```diff
@@ -85,13 +87,33 @@ private ExperimentManager(Submitter submitter) {
    */
   public Experiment createExperiment(ExperimentSpec spec) throws SubmarineRuntimeException {
     checkSpec(spec);
+
+    // Submarine sdk will get experimentID and JDBC URL from environment variables in each worker,
+    // and then log experiment metrics and parameters to submarine server
+    ExperimentId id = generateExperimentId();
+    String url = getSQLalchemyURL();
```
Comment (jiwq) [inline submarine-server/server-core/src/main/java/org/apache/submarine/server/experiment/ExperimentManager.java:94]:
```suggestion
    String url = getSQLAlchemyURL();
```

**Comment by pingsutw** [comment]
@liuxunorg @jiwq Thanks for the review.
I've updated the PR.

File: submarine-server/server-core/src/main/java/org/apache/submarine/server/experiment/ExperimentManager.java [hunk 3]
```diff
@@ -85,13 +87,33 @@ private ExperimentManager(Submitter submitter) {
...
+    ExperimentId id = generateExperimentId();
+    String url = getSQLAlchemyURL();
+    spec.getMeta().getEnvVars().put(RestConstants.JOB_ID, id.toString());
+    spec.getMeta().getEnvVars().put(RestConstants.SUBMARINE_TRACKING_URI, url);
+
     Experiment experiment = submitter.createExperiment(spec);
-    experiment.setExperimentId(generateExperimentId());
+    experiment.setExperimentId(id);
+
+    spec.getMeta().getEnvVars().remove(RestConstants.JOB_ID);
```
Comment (tangzhankun) [inline submarine-server/server-core/src/main/java/org/apache/submarine/server/experiment/ExperimentManager.java:101]: I don't quite understand why put the environment to the spec and then delete them?

Comment (pingsutw) [reply submarine-server/server-core/src/main/java/org/apache/submarine/server/experiment/ExperimentManager.java:101]: Those two environment variables are for internal usage, we set these environment variables in pods.
But we would not like to see them in the experiment spec.

**Pull request closed without merging**